src/config.ts

```diff
@@ -22,9 +22,24 @@ export interface Config {
 
   /**
    * URL of the XHR proxy. If unset, XHRs are not proxied.
+   *
+   * Must be set if templateProxyURL is set.
    */
   xhrProxyURL?: string;
 
+  /**
+   * URL of the template proxy. If unset, template rendering is not proxied.
+   */
+  templateProxyURL?: string;
+
+  /**
+   * Runs some transforming preprocessing modules on the output received from
+   * the template proxy.
+   *
+   * Must be false if templateProxyURL is unset.
```

Comment:
do we enforce this check in the code somewhere?

Reply:
Hm, good point: `validateConfig` does check this, but it seems we're not calling it anywhere right now.